protocol/communities/manager.go

```diff
@@ -1478,107 +1480,90 @@ func (m *Manager) HandleCommunityEventsMessageRejected(signer *ecdsa.PublicKey,
 	return reapplyEventsMessage, nil
 }
 
-func (m *Manager) HandleCommunityPrivilegedUserSyncMessage(signer *ecdsa.PublicKey, message *protobuf.CommunityPrivilegedUserSyncMessage) error {
```

Comment:
I split this function into two functions, one for requests to join and one for the tokens. The reason is that when we receive those messages from other privileges users, we didn't update the client at all. Now, `HandleRequestToJoinPrivilegedUserSyncMessage` returns the requestsToJoin so that we can add them to the Response.